flow-typed/content.js

```diff
@@ -24,7 +24,7 @@ declare type WatchHistory = {
 declare type PlayingUri = {
   uri?: ?string,
```

Comment:
It broke embed in comments.  It now brings you to the claim rather than playing inline.

Reply:
Oh, for the last checkbox, it wasn't playing because the censorship video was in both the Post and Comment.  If already playing in Post, it can't play in Comment unless another embed is played.

Can ignore I guess.